prisma/infra/prisma/schema.prisma

```diff
@@ -0,0 +1,597 @@
+generator client {
```

Comment:
Was this generated 100% by the project or was it edited? Probably better to not include this if the former?

Reply:
Yes. After the enum change there was nothing edit. Prisma has users change this to reflect the sql schema changes, but come think of it in our case since it is all introspected and managed by the modules I think it makes more sense to `.gitignore` it as you are suggesting